Nemt/nemt.cpp

```diff
@@ -15,44 +15,46 @@ Nemt::Nemt(int cl) {
     counter = 0;
 }
 
-int Nemt::giveX(int x) {
-    giveV(x);
+int Nemt::give_x(int block_color) {
+    give_v(block_color);
     return x_point_goal;
 }
 
-int Nemt::giveLeft(int x) {
-    giveV(x);
+int Nemt::give_left(int block_color) {
+    give_v(block_color);
     return left_point_goal;
 }
 
-int Nemt::giveRight(int x) {
-    giveV(x);
+int Nemt::give_right(int block_color) {
+    give_v(block_color);
     return right_point_goal;
 }
 
-void Nemt::giveV(int block_color) {
+void Nemt::give_v(int block_color) {
     blocks = pixy.getBlocks();
-    if(blocks) // if we see at least one objects we can be happy 
-    {
-        for (int i = 0; i < blocks; i++) // runnig through all objects 
-        {
-            // if object i == goal_color and is bigger that captured object that it is object with bigger priority
-            if(pixy.blocks[i].signature == block_color &&
-               pixy.blocks[i].width > pixy.blocks[goal_number].width)
-            { 
-                goal_number = i; // we set color of goal
+
+    if (blocks) {
+        for (int i = 0; i < blocks; i++) {
+            if (pixy.blocks[i].signature == block_color &&
+               pixy.blocks[i].width > pixy.blocks[goal_number].width) {
+                goal_number = i;
             }            
-        } 
+        }
 
         x_point_goal = pixy.blocks[goal_number].x;
-        left_point_goal = pixy.blocks[goal_number].x - pixy.blocks[goal_number].width/2; // left pint of goal // lpg = x - width/2    
-        right_point_goal = pixy.blocks[goal_number].x + pixy.blocks[goal_number].width/2; // right point of goal // rpg = x + width/2   
+
+        left_point_goal = pixy.blocks[goal_number].x
+                - pixy.blocks[goal_number].width/2;
+
+        right_point_goal = pixy.blocks[goal_number].x
+                + pixy.blocks[goal_number].width/2;
+
         counter = 0;
     } else {
-        counter++;
+        counter ++;
```

Comment:
Not what I meant, sorry.

Prefix increment or preincrement is: `++var`, postincrement is: `var++`.

Preincrement is somehow like:

```cpp
int preincrement(int &value) {
    value += 1;
    return value;
}
```

while postincrement is:

```cpp
int postincrement(int &value) {
    int ret = value;
    value += 1;
    return ret;
}
```

So in this case, either remove the space back (to `counter++`) or put it in preincrement form (`++counter`).